web/packages/plugins/dmt-app/src/pages/SearchPage.tsx

```diff
@@ -237,6 +238,7 @@ function FilterContainer({
   resetSearchSettings: () => void
 }) {
   const [attributes, setAttributes] = useState<Array<any>>([])
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore